2019/pt-br/src/0xa9-improper-assets-management.md

```diff
@@ -0,0 +1,54 @@
+API9:2019 Gerenciamento de Ativos Inapropriado
+==============================================
+
+| Agentes/Vetores | Fraquezas de Segurança | Impactos |
+| - | - | - |
+| Específico da API : Explorabilidade **3** | Prevalência **3** : Detecção **2** | Técnico **2** : Específico do negócio |
+| Versões desatualizadas de APIs geralmente carecem de *patches* e são um meio fácil de comprometer sistemas sem a presença de mecanismos de segurança no estado-da-arte, e que podem existir também para proteger versões atualizadas de APIs. | Documentação desatualizada torna mais difícil encontrar e/ou corrigir vulnerabilidades. Falta de um inventário de ativos e estratégias de retirada levam ao cenário de sistemas sem atualização permanecerem em execução, podendo resultado no vazamento de dados sensíveis. É bastante comum encontrar APIs expostas sem necessidade em razão de conceitos modernos como os de microsserviços, que permite que aplicativos sejam lançados independentemente (ex.: computação em nuvem, kubernetes). | Atacantes ter acesso a dados sensíveis e até tomar o controle de servidores por meio de uma velha e desatualizada API conectada ao mesmo banco de dados. |
+
+## A API está vulnerável?
+
+A API pode estar vulnerável se:
+
+* O propósito do *host* da API não for claro, e se não há respostas explícitas para as seguintes questões:
+  * Em qual ambiente está rodando à API? (Ex.: produção, *staging*, teste, desenvolvimento)?
+  * Quem deve ter acesso via rede à API (Ex.: pública, interna, parceiros)?
+  * Em qual versão está a API em execução?
+  * Que tipo de informação acessa a API (Ex.: Dados pessoais sensíveis)?
+  * Qual é o fluxo da informação?
+* Não existe documentação, ou a documentação existente está desatualizada.
+* Não há um plano de retirada para cada versão da API.
+* Inventário de *hosts* não existe ou está desatualizado.
+* Inventário de serviços de integração, seja interna ou de parceiros, não existe ou está desatualizado.
+* Versões antigas da API continuam rodando sem *patches*.
+
+## Cenários de exemplo de ataques
+
+### Cenário #1
+
+Após um redesenho de sua aplicações, um serviço de pesquisa local deixou uma versão antiga da API (`api.someservice.com/v1`) em execução, não protegida, e com acesso ao banco de dados. Enquanto buscava como alvo a última versão do aplicativo, um atacante percebeu o endereço da API (`api.someservice.com/v2`). Substituindo `v2` por  `v1` na URL, o atacante acessou a versão antiga e não protegida, a qual expões dados pessoais sensíveis de mais de 100 milhões de usuários.
```

Comment:
fix
* where it's written "_Após um redesenho de **sua** aplicações (...)_"
* it should be "_Após um redesenho de **suas** aplicações (...)_"

Reply:
fix
* where it's written "_(...) a qual **expões** dados pessoais (...)_"
* it should be "_(...) a qual **expõe** dados pessoais (...)_"